### PR TITLE
Update for beta 8

### PR DESCRIPTION
**Pull request by KyrneDev**
Sorry this one took me awhile, I was away from my computer.

### CODE REVIEW

File: src/Listener/AddClientAssets.php [hunk 1]
```diff
@@ -21,13 +21,13 @@ class AddClientAssets
      */
     public function subscribe(Dispatcher $events)
     {
-        $events->listen(ConfigureWebApp::class, [$this, 'addAssets']);
+        $events->listen(Rendering::class, [$this, 'addAssets']);
     }
 
     /**
      * @param ConfigureClientView $event
```
Comment (franzliedke) [inline src/Listener/AddClientAssets.php]: Please remove the docblock.

File: src/Listener/SaveSuspensionToDatabase.php [hunk 2]
```diff
@@ -41,13 +41,13 @@ public function __construct(SuspendValidator $validator)
...
-        $events->listen(UserWillBeSaved::class, [$this, 'whenUserWillBeSaved']);
+        $events->listen(Saving::class, [$this, 'whenSaving']);
     }
 
     /**
-     * @param UserWillBeSaved $event
+     * @param Saving $event
      */
-    public function whenUserWillBeSaved(UserWillBeSaved $event)
+    public function whenSaving(Saving $event)
```
Comment (franzliedke) [inline src/Listener/SaveSuspensionToDatabase.php]: How about `whenSavingUser`?

**Review by franzliedke — CHANGES_REQUESTED** [review]

**Comment by franzliedke** [comment]
Ping @CDK2020. Can you take care of the requested changes here and in https://github.com/flarum/flarum-ext-akismet/pull/7? Otherwise I will do it, but I wanted to leave the chance to you. ;)

**Comment by KyrneDev** [comment]
Sorry @franzliedke completely missed you message. I've fixed it, but styleCI is never happy

**Comment by franzliedke** [comment]
@CDK2020 Great, thanks for the quick response. Can you also apply my suggestions here, in the flarum-ext-suspend PR?

**Pull request merged**